### PR TITLE
Redirect call for proposal to another page if cfp is closed

### LINKED ISSUE #15: Autoclose CfP
The CfP should be disabled at a given date/time. The CMS page can be disabled by using page publish end, but the form should redirect to a page disabling the submit

### PR DESCRIPTION
**Pull request by EmadMokhtar**
Redirect call for proposal to another page if cfp is closed.
closes #15

### CODE REVIEW

File: djangocon_europe/settings.py [hunk 1]
```diff
@@ -278,3 +278,5 @@
 META_FB_TYPE = 'Article'
 META_GPLUS_SCOPE_CATEGORY = 'Article'
 BLOG_TWITTER_AUTHOR = '@DjangoConEurope'
+
+CFP_CLOSING_DATE = env('CFP_CLOSING_DATE')
```
Comment (yakky) [inline djangocon_europe/settings.py]: A Default must be provided here. I actually don't think we need this from the environment. Can be easily hardcoded in settings

**Review by yakky — CHANGES_REQUESTED** [review]

**Comment by yakky** [comment]
@EmadMokhtar Thanks, it looks really nice. Only minor comment added

File: djangocon_europe/settings.py (hunk 1 above)
Comment (EmadMokhtar) [reply djangocon_europe/settings.py]: @yakky One question, Are users can submit on 8th of Jan. 2017?

Comment (yakky) [reply djangocon_europe/settings.py]: Yes. 2017-01-08 24.00 CET (or 2017-01-09 00.00 CET)

Comment (EmadMokhtar) [reply djangocon_europe/settings.py]: Perfect, Thanks 👍

**Review by yakky — APPROVED** [review]

**Comment by yakky** [comment]
@EmadMokhtar nice, thanks for completing this!

**Pull request merged**